Sources/AppcuesKit/Presentation/Actions/Appcues/AppcuesLinkAction.swift

```diff
@@ -46,12 +46,25 @@ internal class AppcuesLinkAction: ExperienceAction {
                 completion()
             } else {
                 if openExternally {
-                    urlOpener.open(url, options: [:]) { _ in completion() }
+                    openLink(appcues: appcues, completion: completion)
```

Comment:
When I was just thinking about the concept, I wasn't sure how we'd get access to the `navigationDelegate` in the link action. Turns out that's easy 👌

Reply:
@mmaatttt  - maybe this line 49 should continue to just launch the link through the UIApplication (the old urlOpener approach)? And line 55 would be the only line that calls into the `navigationDelegate`. I say this because I realize that at this line, we already know it is an http link (not a scheme link), and it is not a universal link (per line 43), and it's being requested to openExternally. This is just adding extra burden on the host application to conditionally check for web links and open into the browser when we could do it here.

Reply:
True, true. Good catch!